sanitizer/PeliasParameterError.js

```diff
@@ -0,0 +1,15 @@
+class PeliasParameterError extends Error {
+  constructor(message = '') {
+    super(message);
+  }
+
+  toString() {
+    return this.message;
+  }
+
+  toJSON() {
+    return this.message;
+  }
+}
+
+module.exports = PeliasParameterError;
```

Comment:
no unit tests for this new class?

Reply:
No, personally I think that would be overtesting in this case, given the simple implementation. However I am going to add a new test or two in other places that use this class.

Reply:
Haha, yea I hear ya, just don't want any errors in our error handling code! :)

Reply:
Or more specifically, we *do* want an error, this error :)